packages/@okta/vuepress-site/docs/guides/build-sso-integration/create-your-app/openidconnect/create.md

```diff
@@ -3,6 +3,7 @@
 4. On the Create a new app integration page, select **OpenID Connect** in the **Sign-in method** section.
 1. Choose either **Web Application** or **Single-Page Application** as the **Application type** for your integration. The OIN only publishes cloud-based SaaS apps, either traditional Web applications with a back-end or a modern browser-based SPA.
 1. Click **Next**.
-1. On the **General Settings** tab, enter a name for your integration and (optionally) upload a logo.
-1. On the **Configure OpenID Connect** tab, add your **Sign-in redirect URIs** and optional **Sign-out redirect URIs**. These URIs must be [absolute URIs](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#URI_resolution), and you can specify more than one.
+1. In **General Settings**, enter a name for your integration and (optionally) upload a logo.
+1. Add your **Sign-in redirect URIs** and optional **Sign-out redirect URIs**. These URIs must be [absolute URIs](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#URI_resolution), and you can specify more than one.
+1. In **Assignments**, assign a group or leave the **Everyone** default. Be sure to verify that the users you want to have access are assigned to the group that you select. For instructions on how to assign the application to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
```

Comment:
... how to assign the app integration to individual users ...

Reply:
Fixed